bin/qds.py

```diff
@@ -390,6 +393,29 @@ def clustermain(args):
         usage()
     return globals()["cluster_" + action + "_action"](clusterclass, args)
 
+def account_create_action(accountclass, args):
+    arguments = accountclass._parse_account(args, action = "create")
+    if arguments is not None:
+        result = accountclass.create(**arguments)
+    return 0
+
+
+
+def accountmain(args):
+    accountclass = Account
+    actionset = set(["create"])
```

Comment:
Please don't do it like this. For all new endpoints we are following the approach used in `reportmain` or `schedulemain` etc. where all the parsing is left to Argparse.